private/bufpkg/bufmodule/bufmodule.go

```diff
@@ -110,6 +111,10 @@ type Module interface {
 	//
 	// This may be nil, since older versions of the module would not have this stored.
 	LintConfig() *buflintconfig.Config
+	// Manifest returns the manifest for the module (possibly empty).
+	Manifest() manifest.Manifest
+	// Blobs return the raw data for the module (possibly empty).
+	Blobs() manifest.BlobSet
```

Comment:
nit: if we keep this, this should be named `BlobSet()`

Reply:
Done.